_data/banking.yml

```diff
@@ -598,6 +598,13 @@ websites:
       - hardware
     doc: https://www.intesasanpaolo.com/it/common/footer/sicurezza.html
 
+  - name: Jovia Financial
+    url: https://www.joviafinancial.com/
+    img: jovia.png
+    tfa:
+      - sms
+      - phone
```

Comment:
uhm, no documentation?

Reply:
Not really, I'm afraid. There is
> https://www.joviafinancial.com/knowledge-base/if-i-am-traveling-internationally-can-i-receive-a-one-time-passcode-via-sms-text-or-automated-voice-call-to-access-online-banking

but that's rather "second hand documentation".

Reply:
At least it proves that they have 2FA